myhpi/static/scss/footer.scss

```diff
@@ -1,5 +1,5 @@
 .footer-myhpi {
-    background-color: var(--bs-gray-700);
+    background-color: var(--bs-gray-800);
 
     .page-content {
         padding-top: 1.5rem;
```

Comment:
A difference between the menu and background color would be helpful to differentiate both better (now they flow into each other):

![grafik](https://github.com/user-attachments/assets/11399458-0ee3-4bf9-91aa-4a6a37e75ac7)

Before:

![grafik](https://github.com/user-attachments/assets/f172c875-3eaa-41b5-a30f-dacffc807302)

Especially helpful in the dark mode:

![grafik](https://github.com/user-attachments/assets/efc6566f-8880-4fe6-8cd8-f208a16307eb)